integration/config/orderer.yaml

```diff
@@ -77,24 +68,22 @@ General:
         ServerCertificate:
         # ServerPrivateKey defines the file location of the private key of the TLS certificate.
         ServerPrivateKey:
-    # Genesis method: The method by which the genesis block for the orderer
-    # system channel is specified. Available options are "provisional", "file":
-    #  - provisional: Utilizes a genesis profile, specified by GenesisProfile,
-    #                 to dynamically generate a new genesis block.
-    #  - file: Uses the file provided by GenesisFile as the genesis block.
-    GenesisMethod: provisional
-
-    # Genesis profile: The profile to use to dynamically generate the genesis
-    # block to use when initializing the orderer system channel and
-    # GenesisMethod is set to "provisional". See the configtx.yaml file for the
-    # descriptions of the available profiles. Ignored if GenesisMethod is set to
-    # "file".
-    GenesisProfile: SampleInsecureSolo
-
-    # Genesis file: The file containing the genesis block to use when
-    # initializing the orderer system channel and GenesisMethod is set to
-    # "file". Ignored if GenesisMethod is set to "provisional".
-    GenesisFile: genesisblock
+
+    # Bootstrap method: The method by which to obtain the bootstrap block
+    # system channel is specified. The option can be one of:
+    #   "file" - path to a faile containing the genesis block or config block of system channel
+    #   "none" - allows an orderer to start without a system channel configuration
+    BootstrapMethod: file
+    #BootstrapMethod: none
```

Comment:
Why did we add this (commented) out line?  Tricked me first in thinking that we changed something from the standard fabric-samples version when we actually didn't.

BTW: we didn't do that in that past (although it would have been good :-) but maybe should do it now: Add a comment at the top of this file (as well as core.yaml and other files we derived from standard fabric files) that this is derived from `config/orderer.yaml` from `fabric-samples.git`. This should help in maintaining these files ....
In fact, only looking closer i realized we started originally from `sampleconfig` in `fabric.git` which also contained the `msp` files => we should also update `msp/config.yaml` to contain the v2 changes ...